department-of-veterans-affairs/content-build
# 9755 add qa to support detail page

## Description
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9901

Adds Q&A component to the support_resources_detail_page template

## Testing done
Local testing with test data in tugboat environment.

Steps to test:
- Rebuild local to tugboat instance: https://main-vpsyfavqes4gp7mmxayff3gvxqsntwzz.demo.cms.va.gov/
- navigate to resource and support page: resources/getting-a-gi-bill-extension/
- Observe that data payload reflects new graphQL query and that Q&A content block data is included
- Observe that Q&A sections are rendered on page

## Screenshots
<img width="827" alt="Screen Shot 2022-08-24 at 9 27 43 AM" src="https://user-images.githubusercontent.com/61624970/186431814-8d797444-e45d-4826-9763-184248fdad16.png">
<img width="894" alt="Screen Shot 2022-08-24 at 9 29 30 AM" src="https://user-images.githubusercontent.com/61624970/186431828-71bcdc29-5f70-423e-9610-f669ab3f394a.png">
<img width="1011" alt="Screen Shot 2022-08-24 at 9 28 29 AM" src="https://user-images.githubusercontent.com/61624970/186431838-bd4512b7-f573-4d3d-b649-2adbeceff5f0.png">


## Acceptance criteria
- [ ] Q&As render on on support resource detail page type pages when data is provided

## Definition of done
